sklearn/svm/tests/test_svm.py

```diff
@@ -353,6 +353,7 @@ def test_decision_function_shape():
     assert_equal(dec.shape, (len(X_train), 10))
 
 
+@ignore_warnings
```

Comment:
Yes, I think that would be good. Just use reg.predict instead of ``decision_function`` below, and then compare the predict and the decision_function in a catch_warnings block.